doc/src/appendices/suiterc-config-ref.rst

```diff
@@ -956,8 +953,8 @@ Allows tasks to spawn out to ``max active cycle points``
 submitted before its successor can be spawned.
 
 *Important*: This should be used with care given the potential impact of
-additional task proxies both in terms of memory and cpu for the cylc daemon as
-well as overheads in rendering all the additional tasks in gcylc. Also, use
+additional task proxies in terms of memory and cpu for the cylc daemon.
```

Comment:
Not a porting issue, but might as well update *cylc daemon* to *suite server program* or *workflow server program*.

Reply:
Art thou requesting these changes here? Or do you want a new ticket?

Reply:
Either way. But given that the line is already touched, it is probably more convenient to incorporate.